socorro/unittest/processor/test_breakpad_transform_rules.py

```diff
@@ -632,20 +488,39 @@ def test_stackwalker_fails_2(self, mocked_subprocess_module):
         assert processed_crash.mdsw_return_code == -1
         assert processed_crash.mdsw_status_string == "unknown error"
         assert not processed_crash.success
+        command_line = config.command_line.format(
+            **dict(
```

Comment:
This seems kinda weird vs just passing the kwargs normally, with `**config` as the last kwarg. It's trying to be defensive against a name collision with the config options, isn't it? That seems kinda unnecessary, but that might just be me.

I notice that this is copied from the logic in `_action`. Should we simplify this in both spots?

Reply:
Yeah. I copied it without thinking a lot about it, but you're right--this is junk. I'll redo it in both places.